docs/components/User/services/canonicalizing.rst

```diff
@@ -20,7 +20,7 @@ letters. Example usage:
     // directory, relative to this file
     require_once __DIR__.'/../vendor/autoload.php';
 
-    use Sylius\Component\User\Model\Customer;
+    use Sylius\Component\Customer\Model\Customer;
```

Comment:
There is no `Customer` in User component anymore, so we should update the whole snippet and use only `User` :)